db/migrate/20191106210313_add_location_fields_to_organization.rb

```diff
@@ -0,0 +1,12 @@
+class AddLocationFieldsToOrganization < ActiveRecord::Migration
+  def change
+    change_table :organizations do |t|
```

Comment:
Is this syntax roll-backable?

Reply:
It is — can't find it in the docs but simple enough to verify

Reply:
Cool, I learn new things every day!